.travis.yml

```diff
@@ -1,25 +1,12 @@
 language: c
 sudo: required
+service: docker
 install:
-  - wget https://raw.githubusercontent.com/ocaml/ocaml-travisci-skeleton/master/.travis-docker.sh
-  - wget https://raw.githubusercontent.com/xapi-project/xapi-travis-scripts/master/coverage.sh
-script:
-  - bash -ex .travis-docker.sh
-services:
-  - docker
+  - wget https://raw.githubusercontent.com/ocaml/ocaml-ci-scripts/master/.travis-docker.sh
+  - wget https://raw.githubusercontent.com/xapi-project/xs-opam/master/tools/xs-opam-ci.env
+  - source xs-opam-ci.env
+script: bash -ex .travis-docker.sh
 env:
   global:
-    - OCAML_VERSION=4.07
     - PINS="xen-api-client:. xen-api-client-lwt:. xen-api-client-async:."
     - PACKAGE=xen-api-client
-    - DISTRO="debian-9"
-  matrix:
-    - BASE_REMOTE=git://github.com/xapi-project/xs-opam \
-        POST_INSTALL_HOOK="env TRAVIS=$TRAVIS TRAVIS_JOB_ID=$TRAVIS_JOB_ID bash -ex coverage.sh"
-    - PACKAGE=xen-api-client-async BASE_REMOTE=git://github.com/xapi-project/xs-opam POST_INSTALL_HOOK="make async-examples"
-    - PACKAGE=xen-api-client-lwt BASE_REMOTE=git://github.com/xapi-project/xs-opam POST_INSTALL_HOOL="make lwt-examples"
```

Comment:
Please keep these two jobs (even though the lwt was broken because of a typoe in `INSTALL_HOOL`)